### PR TITLE
Added launch files for navstack code + mobileIO python script

### PR DESCRIPTION
**Pull request by stmian**
This pull request added 3 types of files to the repo:

3 launch files to use with the new omnibase_odometry node
Drive by wire (keyboard), drive using Mobile IO, and Gmapping

An updated Python script that allows for the omnibase node to be driven using the Mobile IO interface

A saved RViz configuration used by Rosie for gmapping visualization

### CODE REVIEW

File: hebiros_advanced_examples/launch/omnibase/omnibase_odometry_navigation.launch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<launch>
+    <node name="hebiros_node" pkg="hebiros" type="hebiros_node" respawn="false" output="screen"/>
+
+    <node pkg="hebiros_advanced_examples" type="example_omnibase_odometry" name="example_omnibase_odometry" />
+
+    <node pkg="hebiros_advanced_examples" type="example_omnibase_free_roam" name="example_omnibase_free_roam" />
+
+    <node pkg="hebiros_advanced_examples" type="key_read.py" name="key_read" />
+
+  </launch>
```
Comment (iamtesch) [inline hebiros_advanced_examples/launch/omnibase/omnibase_odometry_navigation.launch]: How is this different from https://github.com/HebiRobotics/HEBI-ROS/blob/develop/hebiros_advanced_examples/launch/omnibase/example_omnibase_odometry.launch ?

File: hebiros_advanced_examples/launch/omnibase/rosie_gmapping.launch [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<launch>
```
Comment (iamtesch) [inline hebiros_advanced_examples/launch/omnibase/rosie_gmapping.launch]: I would go ahead and hold off on adding this file until we get all the dependent files in and cleaned up (e.g., the .xacro) so we can have this as a documented + functioning demo

File: hebiros_advanced_examples/src/rviz/rosie_gazebo_model.rviz [hunk 3]
```diff
@@ -0,0 +1,287 @@
+Panels:
```
Comment (iamtesch) [inline hebiros_advanced_examples/src/rviz/rosie_gazebo_model.rviz]: I would prefer to go ahead and hold off on adding this until we had something that referenced it as well.

**Comment by stmian** [comment]
removed extraneous launch files, and updated mobile_io

**Pull request merged**